packages/core/android/src/main/java/com/datadog/reactnative/DdSdk.kt

```diff
@@ -6,31 +6,52 @@
 
 package com.datadog.reactnative
 
-import com.datadog.android.bridge.DdBridge
-import com.datadog.android.bridge.DdSdk as SDKDdSdk
+import android.content.Context
+import android.util.Log
+import com.datadog.android.DatadogSite
+import com.datadog.android.core.configuration.Configuration
+import com.datadog.android.core.configuration.Credentials
+import com.datadog.android.privacy.TrackingConsent
+import com.datadog.android.rum.RumMonitor
+import com.datadog.android.rum.tracking.ActivityViewTrackingStrategy
 import com.facebook.react.bridge.Promise
 import com.facebook.react.bridge.ReactApplicationContext
 import com.facebook.react.bridge.ReactContextBaseJavaModule
 import com.facebook.react.bridge.ReactMethod
 import com.facebook.react.bridge.ReadableArray
 import com.facebook.react.bridge.ReadableMap
+import java.net.InetSocketAddress
+import java.net.Proxy
+import java.util.Locale
 
 /**
  * The entry point to initialize Datadog's features.
  */
-class DdSdk(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
+class DdSdk(reactContext: ReactApplicationContext, private val datadog: DatadogSDKWrapper = DatadogSDKWrapper()) : ReactContextBaseJavaModule(reactContext) {
```

Comment:
I don't see the unit tests here. Now this class has quite some logic so it will need related unit tests.

Reply:
Good point, I'm planning to move the unit tests over from the bridges as well

Reply:
minor: this line is too long, it should be wrapped. Ideally we need to move the checks like `ktLint` and `detekt` from bridges as well and add them to the CI process.